UIMentoringSession/MenteesChallenges/Pets Main Card UI Chellenge/YefimYakunin/PetCollectionCell/YeYa_PetCell.swift

```diff
@@ -0,0 +1,33 @@
+import UIKit
+
+class YeYa_PetCell: UICollectionViewCell {
```

Comment:
make it final

Reply:
final class YeYa_PetCell: UICollectionViewCell {

Added final